requirements.txt

```diff
@@ -0,0 +1,9 @@
+annotated-types==0.7.0
+certifi==2025.1.31
```

Comment:
The `certifi` package has a future date version (`2025.1.31`) which doesn't exist yet. This will cause installation failures as pip cannot find this non-existent version. It should be changed to the latest actual release.